core/templates/pages/profile-page/profile-page.component.ts

```diff
@@ -101,80 +101,82 @@ export class ProfilePageComponent {
     this.DEFAULT_PROFILE_PICTURE_URL = this.urlInterpolationService
       .getStaticImageUrl('/general/no_profile_picture.png');
     this.loaderService.showLoadingScreen('Loading');
-    this.profilePageBackendApiService.fetchProfileDataAsync()
-      .then((data) => {
-        this.data = data;
-        this.username = {
-          title: 'Username',
-          value: data.usernameOfViewedProfile,
-          helpText: data.usernameOfViewedProfile
-        };
-        this.usernameIsLong = data.usernameOfViewedProfile.length > 16;
-        this.userBio = data.userBio;
-        this.userDisplayedStatistics = [{
-          title: 'Impact',
-          value: data.userImpactScore,
-          helpText: (
-            'A rough measure of the impact of explorations created by ' +
-            'this user. Better ratings and more playthroughs improve ' +
-            'this score.')
-        }, {
-          title: 'Created',
-          value: data.createdExpSummaries.length,
-          helpText: null
-        }, {
-          title: 'Edited',
-          value: data.createdExpSummaries.length,
-          helpText: null
-        }];
+    Promise.all([
+      this.profilePageBackendApiService.fetchProfileDataAsync(),
+      this.profilePageBackendApiService.fetchProfileImageDataUrlAsync()
+    ]).then(([data, profileImageData]) => {
```

Comment:
I assumed that the new `Profile Data` construct in the backend storage will no longer include `Profile Photo` data.

Reply:
@lkbhitesh07 @vojtechjelinek can you comment on this?

Reply:
Yes it won't contain the profile photo data, the profile photo will be available just through an URL targeting directly the GCS.